pkgs/development/interpreters/io/default.nix

```diff
@@ -32,6 +32,12 @@ stdenv.mkDerivation {
     sed -ie \
           "s/add_subdirectory(addons)/#add_subdirectory(addons)/g" \
           CMakeLists.txt
+    # glibc 2.32 removed sysctl.h
+    sed -i '/sys\/sysctl.h/g' libs/iovm/source/IoSystem.c
```

Comment:
Please replace with fetchpatch: https://github.com/IoLanguage/io/commit/6d9cdf57b54046a3470f8059618c92bf2f913183.patch
also see: https://github.com/IoLanguage/io/pull/445

Reply:
That patch did not fix the Issue.  
To wrap things up i am fetching my own patch https://github.com/alwinber/io/commit/e977b374eea77f9d5632463ca688b1e1f5b03f82.patch
It is based on this comment https://github.com/IoLanguage/io/pull/445#discussion_r581458969
I did not want to create a duplicate PR upstream.

Reply:
I closed my pull request instead.